docs/source/technical_tutorials/search/maap_stac_r.ipynb

```diff
@@ -0,0 +1,734 @@
+{
```

Comment:
<p>For the purposes of this notebook I don't think we need the <code>if(!is.null(collections$collections))</code> check since we know that this STAC API has collections. Maybe it would be better to show how to retrieve the metadata for a specific collection given an ID.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/464/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Often people will use the tutorials as copy-paste example code. So if we expect people to use this notebook that way, I’d suggest leaving in the control-flow/checks on the collections. It’s also implicitly a good lesson that you can have empty Collections and that they need to be handled.

If we don’t expect anyone to copy/paste the code from this notebook, then I agree we could leave that out.

Reply:
<p>Since my knowledge of R is minimal, I could very well be wrong, but I suspect that <code>!is.null(collections$collections)</code> will <em>always</em> be true because <code>collections$collections</code> will <em>always</em> be a list. Of course, the list <em>could</em> be empty, but it should never be <code>NULL</code>. I suspect what you might want here is <code>length(collections$collections) &gt; 0</code> instead. It appears to me that there are a number of places where a null check is being performed, but a length check should be performed instead.</p><p> </p><p>Again, I could be totally off base, so someone please correct me if I'm wrong here.</p>

Reply:
You might be right @chuckwondo 
We need to test against an endpoint with 0 collections, perhaps it will be easier to find a 0 items collection to test this on.

Reply:
<p>Tested with a collection that had 0 items and y'all were right - we updated to do link checks!</p>